src/WireMock.Net/Serialization/MatcherMapper.cs

```diff
@@ -101,7 +101,8 @@ public MatcherMapper(WireMockServerSettings settings)
                 return new JmesPathMatcher(matchBehaviour, matchOperator, stringPatterns);
 
             case nameof(XPathMatcher):
-                return new XPathMatcher(matchBehaviour, matchOperator, stringPatterns);
+                var xmlNamespaces = matcher.XmlNamespaceMap;

```

Comment:
For this new functionality on the XPtahMAtcher:

The Map method : `public MatcherModel? Map(IMatcher? matcher)` should also be updated to convert an IMatcher to a MatcherModel so that when you retrieve the mappings as json, these these XmlNamespaceMap values are returned.


Also a Unit test for this should be added

Reply:
I hope that I got - I will take a look at the code change. And yes, a unit test then ;-)

Reply:
Implemented and unit test written